packages/openneuro-components/src/form/Radio.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import './forms.scss'
+
+export interface RadioProps {
+  elementId?: string
+  label: string
+  radioGroupName: string
+  radioNumber: string
+  onClick?: () => void
+  checked: boolean
+}
+
+export const Radio: React.FC<RadioProps> = ({
+  elementId,
+  label,
+  radioGroupName,
+  radioNumber,
+  onClick,
+  checked,
+  ...props
+}) => {
+  return (
+    <span className="custom-radio">
+      <input
+        type="radio"
+        id={elementId}
+        name={radioGroupName}
+        checked={checked}
+        onClick={onClick}
+        value={label}
+      />
+      <label htmlFor={elementId}>{label}</label>
```

Comment:
It looks like the `elementId` prop is just used for the `htmlFor` attribute. If that's the case, I don't think we need an `id` prop, and can make a unique id with something like `const id = name + label` and then use that